staff-guidelines.md

```diff
@@ -32,3 +32,23 @@ Following social contract should be one of the considerations of “performance
   - Initiatives (e.g. Gamejams, TOPathons, Fireside chats)
   - Etc.
 - Communications should happen on a semi-regular cadence
+
+---

```

Comment:
Minor and outside of the scope of this but the headings in this doc are a little wonky, the h1s should probably be h2s

Reply:
Agreed! 

[8270377](https://github.com/TheOdinProject/top-meta/pull/186/commits/82703773e8d004c679e3d5dbcd725589b7d1cce1)

Reply:
Unfortunately the PR is going to be much harder to read now, because there was also a lot of white space at the end of a lot of lines, so I went ahead and removed them as I fixed the headings.